PyTado/interface/interface.py

```diff
@@ -77,6 +77,11 @@ def getZones(self):
         """Gets zones information. (deprecated)"""
         return self.get_zones()
 
+    @deprecated("set_child_lock")
+    def setChildLock(self, device_id, enabled):
+        """Set the child lock for a device"""
+        return self.set_child_lock(device_id, enabled)
+
```

Comment:
do we need to add deprecated methods if they haven't been there yet? What's the advantage? I mean, if somebody needs the new method, he could also change his api to not use deprecated methods anymore.

Reply:
True, I didn't went lazy and went for full support still. Extra service. :)